debian/gen_changelog

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+# Create the Tvheadend changelog
```

Comment:
Hi @Jalle19 ,

This line requires to change the name to `# Create the ***minisatip*** changelog`.
This is not the Tvheadend project!

Reply:
Thanks, fixed